### PR TITLE
MAUI-886932 UG documentation for Activation mode, Maximum Zoom level, Inside axis element and Zooming & panning event support.

### PR DESCRIPTION
**Pull request by Harsha-SF4223**
**Description**

- Prepared the UG document for activation mode.
- Added UG documentation for positioning the labels and ticks in the axis of Cartesian Chart.
- Added UG for Events in Zooming and panning in Cartesian Chart.
- Added UG documentation for Positioning the labels and ticks in axis of Polar Chart.
- Added the Maximum zoom level UG.
- Changed image path address in Stacked Area 100.
- Changed name from apperance to appearance in trackball file and customised to customized.
Reference

**Reference**

Xamarin:
- Activation mode - https://help.syncfusion.com/xamarin/charts/trackball#activation-mode

Flutter: 
- Activation mode - https://help.syncfusion.com/flutter/cartesian-charts/trackball-crosshair#activation-mode

### CODE REVIEW

**Comment by SaiyathAliFathima** [comment]
@Harsha-SF4223  , update the PR title , include the maximum zoom level changes

**Comment by Harsha-SF4223** [comment]
> @Harsha-SF4223 , update the PR title , include the maximum zoom level , inside axis element , zoom pan event changes

I have updated the title

File: MAUI/Cartesian-Charts/Trackball.md [hunk 1]
```diff
@@ -57,31 +57,69 @@ The [DisplayMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.Cha
...
+* `None` – Hides the visibility of trackball when setting activation mode to `None`. It will be activated when calling 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+      ...
+    <chart:SfCartesianChart.TrackballBehavior>
+        <chart:ChartTrackballBehavior ActivationMode = "LongPress"/>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Trackball.md:101]: @Harsha-SF4223  ,As its default value is "LongPress,"  why do we need to explain the code snippet?

File: MAUI/Polar-Charts/Axis/Axislabels.md [hunk 2]
```diff
@@ -11,6 +11,44 @@ documentation: ug
 
 Axis labels are used to display the units, measures, or category values of an axis in a user-friendly way. They are generated based on the range and the values bound to the [XBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_XBindingPath) or [YBindingPath](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.XYDataSeries.html#Syncfusion_Maui_Charts_XYDataSeries_YBindingPath) properties of the series.
 
+## Positioning the labels
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Axis/Axislabels.md:14]: @Harsha-SF4223 ,As this feature has been moved in the SP release, could you please update the API reference for the inside axis elements?

File: MAUI/Polar-Charts/Axis/Ticklines.md [hunk 3]
```diff
@@ -49,6 +49,43 @@ chart.SecondaryAxis = secondaryAxis;
 
 {% endtabs %}
 
+## Positioning the ticks
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Axis/Ticklines.md:52]: @Harsha-SF4223 ,As this feature has been moved in the SP release, could you please update the API reference for the inside axis elements?

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Trackball.md (hunk 1 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/Trackball.md:101]: Based on platform, the default values changes. 
In Windows and Mac, the default value is TouchMove.
In Android and iOS, the default value is LongPress.
So, irrespective of the platform, I have assigned the value for activation mode as LongPress.

File: MAUI/Cartesian-Charts/Axis/Ticklines.md [hunk 4]
```diff
@@ -50,6 +50,48 @@ chart.YAxes.Add(secondaryAxis);
 
 {% endtabs %}
 
+## Positioning the ticks
+
+The tick lines can be positioned inside or outside the chart area using the [TickPosition](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_TickPosition) property. [LabelsPosition](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_LabelsPosition) property default value is `AxisElementPosition.Outside`.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Axis/Ticklines.md:55]: Update the API as  Tick Position
image

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

**Review by Saravanan-Madhesh — APPROVED** [review]
Fine

**Review by SaiyathAliFathima — APPROVED** [review]

**Pull request merged**